modules/service/src/main/resources/sparql.conf

```diff
@@ -45,5 +45,8 @@ app {
     # Default pagination information
     page-from = 0
     page-size = 20
+    page-size = ${?DEAFULT_SEARCH_SIZE}
```

Comment:
typo

I would also suggest renaming to: `PAGINATION_DEFAULT_SIZE` and `PAGINATION_MAX_SIZE` respectively.